src/app/GitExtUtils/GitUI/Interops/COMBOBOXINFO.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Runtime.InteropServices;
+
+namespace System;
+
+internal static partial class NativeMethods
+{
+    [StructLayout(LayoutKind.Sequential)]
+    internal struct COMBOBOXINFO
+    {
+        public int cbSize;
+        public RECT rcItem;
+        public RECT rcButton;
+        public ComboBoxButtonState buttonState;
+        public IntPtr hwndCombo;
+        public IntPtr hwndEdit;
+        public IntPtr hwndList;
```

Comment:
```suggestion
        public uint cbSize;
        public RECT rcItem;
        public RECT rcButton;
        public ComboBoxButtonState buttonState;
        public nuint hwndCombo;
        public nuint hwndEdit;
        public nuint hwndList;
```

`DWORD` maps to `uint`, see https://learn.microsoft.com/dotnet/standard/native-interop/best-practices.
`HWND` maps to `UIntPtr`, and `nuint` is a new better alias for `UIntPrt`.

Reply:
This is really not my field.. Are you sure about `HWND` mapping to `UIntPtr` and not `IntPtr`? https://learn.microsoft.com/en-us/windows/apps/develop/ui-input/retrieve-hwnd
![image](https://github.com/user-attachments/assets/d56de0a9-05c4-4575-9b6f-4740aa0b5985)

Reply:
Yes, I'm sure. The link that I shared contains all the relevant information:
![image](https://github.com/user-attachments/assets/69ecfa18-ae82-4af5-ac25-a3a3f9aec162)

Reply:
This table should not be read in rows, there are two columns of related types and not a mapping between column A and column B
![image](https://github.com/user-attachments/assets/2c632c65-91e8-4d71-b0ad-c7c556f47d7a)

Reply:
Yes, you're right. I'm having a brainfart moment....
HWND == IntPtr == nint.